tests/phpunit/tests/admin/includes/class-wp-upgrader-testcase.php

```diff
@@ -0,0 +1,294 @@
+<?php
+
+/**
+ * Test case class for WP_Upgrader.
+ */
+abstract class WP_Upgrader_TestCase extends WP_UnitTestCase {
+	/**
+	 * Holds error data.
+	 *
+	 * @var array
+	 */
+	protected $error_data = array();
+
+	/**
+	 * Holds plugin data.
+	 *
+	 * @var array
+	 */
+	protected $plugin = array();
+
+	/**
+	 * Holds theme data.
+	 *
+	 * @var array
+	 */
+	protected $theme = array();
+
+	/**
+	 * Whether this is a theme.
+	 *
+	 * @var bool
+	 */
+	protected $is_theme = false;
+
+	/**
+	 * Holds the theme data provider.
+	 *
+	 * @var Theme_Upgrader_Data_Provider
+	 */
+	protected $theme_data_provider;
+
+	/**
+	 * Holds the plugin data provider.
+	 *
+	 * @var Theme_Upgrader_Data_Provider
+	 */
+	protected $plugin_data_provider;
+
+	/**
+	 * Holds the theme directory path.
+	 *
+	 * @var string
+	 */
+	protected static $theme_dir;
+
+	/**
+	 * Holds the original values of the
+	 * 'update_themes' and 'update_plugins' transients.
+	 *
+	 * @var array
+	 */
+	protected static $originals;
+
+	/**
+	 * Sets up the `theme_dir` and `originals` properties before any tests run.
+	 */
+	public static function set_up_before_class() {
+		self::$theme_dir = WP_CONTENT_DIR . '/themes';
```

Comment:
This method doesn't call `parent::set_up_before_class()` - Is there a reason for this, or is this a mistake in PR 850?